core-rust/state-manager/src/staging/cache.rs

```diff
@@ -287,7 +316,10 @@ impl<K, N> AccuTreeDiff<K, N> {
 
 #[derive(Clone)]
 pub struct ImmutableStore {
+    // TODO(engine-merge):: remove
```

Comment:
Can we just use one?

After merging latest scrypto, we can probably just have `ImmutableOrdMap<DbPartitionKey, DbSortKey>` :)